build/stf-run-ci/tasks/main.yml

```diff
@@ -79,6 +79,15 @@
     cmd: "{{ base_dir }}/get_operator_sdk.sh {{ operator_sdk_v1 }}"
     creates: "{{ base_dir }}/working/operator-sdk-{{ operator_sdk_v1 }}"
 
+- name: Set logfile_dir
+  when: not (logfile_dir is defined)
+  ansible.builtin.set_fact:
+    logfile_dir: "{{ base_dir }}/working/logs"
+
+- name: Make sure the logging dir exists
+  ansible.builtin.shell:
+    cmd: mkdir -p {{ logfile_dir }}
```

Comment:
This could also add a `creates` which will help in the future if we enable other lint testing which will complain here about idempotency. Also what about just setting a default path instead of setting it inline?

Reply:
We can't set a default in defaults/main, because the default value is to be relative to base_dir.

That's why there's a "set if not defined"